### DISCUSSION TITLE
v0.9.1

### DISCUSSION THREAD

**Discussion by laurentabbal** [edupyter/EDUPYTER39#1] [Announcements]
Apps : Python 3.9.9 / PIP 21.3.1 / JupyterLab 3.2.5 / Jupyter Notebook 6.4.6 / Thonny 3.3.14
Paquets : Matplotlib 3.5.1 /  Pandas 1.3.4 / Numpy 1.21.4 /  Pillow 8.4.0 / Requests 2.26.0

Téléchargement : https://github.com/edupyter/EDUPYTER/releases/download/v0.9.1/EDUPYTER-39-0.9.1-setup.exe

Cette version 0.9.1 est une version alpha qui n'a été testée que sur un seul environnement. De nombreux autres tests sont nécessaires.
Plus il y aura de testeurs et de retours et plus vite il sera possible de sortir la version alpha, puis bêta.

Toute proposition d'amélioration, de modification ou d'ajout de fonctionnalité est la bienvenue.

<hr /><em>This discussion was created from the release <a href='https://github.com/edupyter/EDUPYTER/releases/tag/'>v0.9.1</a>.</em>

**Comment by nsi-xyz** [comment]
**Installé sur windows 11 :** 

<img width="404" alt="image" src="https://user-images.githubusercontent.com/43805786/145705693-2ba5c4ef-1c8b-45a5-af81-035efcc86cc3.png">
puis
<img width="406" alt="image" src="https://user-images.githubusercontent.com/43805786/145705702-60f54d2f-ecf3-4e8a-b5aa-31630783fdf3.png">

puis ça s'installe.

1. Le raccourci "Thonny" installe un 2ème Thonny ou lance l'existant ? Chez moi c'est mon thonny s'ouvre
2. file:///I:/EDUPYTER-39-0.9.1/loading.html tourne depuis 42 secondes et affiche juste "Le lancement peut prendre quelques secondes... Je l'ai fermé.
3. http://127.0.0.1:8889/lab c'est lancé par contre.

<img width="305" alt="image" src="https://user-images.githubusercontent.com/43805786/145705913-fa9955b6-a0f8-491e-ae80-8bd4baf1f083.png">

**Questions naïve d'ordre pédagogique :** 
1. Comment tu "distribues" un fichier .ipynb aux élèves ?
_Je vois bien la puissance du l'outil mais il me manque un élément pratique_
2. Comment les élèves conservent une trace de leurs fichiers .ipynb ?

Merci en tout cas,

**Reply by laurentabbal** [reply]
Pour les messages d'alerte de Windows, c'est normal et je ne peux pas y faire grand chose. Il faut attendre que l'application soit plus connue.

> Le raccourci "Thonny" installe un 2ème Thonny ou lance l'existant ? Chez moi c'est mon thonny s'ouvre

Normalement, c'est le Thonny d'Edupyter qui doit s'ouvrir. Le problème doit venir du fait que ton Thonny est dans ton PATH. Je vais modifier le mécanisme de lancement et l'ordre des variables.

> file:///I:/EDUPYTER-39-0.9.1/loading.html tourne depuis 42 secondes et affiche juste "Le lancement peut prendre quelques secondes... Je l'ai fermé.

Là aussi c'est normal. C'est une page temporaire qu'il faut fermer à la main. C'est une ruse un peu moche qui a pour but d'avertir et de faire patienter l'utilisateur car Jupyter met pas mal de temps à s'ouvrir.

> Comment tu "distribues" un fichier .ipynb aux élèves ?

Dans mon établissement on fonctionne avec Google Edu Workspace. On utilise donc Classroom pour distribuer les documents et Colaboratory travailler avec les notebooks.

> Comment les élèves conservent une trace de leurs fichiers .ipynb ?

Avec Edupyter, les notebooks s'enregistrent dans le dossier par défaut "Notebooks". Si les élèves veulent garder leurs documents et les utiliser sur d'autres postes il faut qu'ils les sauvegardent dans un "Drive" ou sur leur espace personnel.

**Reply by nsi-xyz** [reply]
Merci pour ces réponses rapides, on utilise aussi Google Edu Workspace donc je m'interrogeais juste s'il existait une autre procédure, simple et efficace, sans Google Classroom.

> Là aussi c'est normal. C'est une page temporaire qu'il faut fermer à la main. C'est une ruse un peu moche qui a pour but d'avertir et de faire patienter l'utilisateur car Jupyter met pas mal de temps à s'ouvrir.

Il faut juste changer le message pour expliquer cela, car on attend qu'il se passe un truc.

> Avec Edupyter, les notebooks s'enregistrent dans le dossier par défaut "Notebooks". Si les élèves veulent garder leurs documents et les utiliser sur d'autres postes il faut qu'ils les sauvegardent dans un "Drive" ou sur leur espace personnel.

Ce qui est logique :-)

**Comment by charlesyo** [comment]
Installation réussie.
Mais rien ne se lance 
Windows 10 Professionnel

**Reply by nsi-xyz** [reply]
Et ceci est affiché ?
EDUPYTER-39-0.9.1/help.html
![image](https://user-images.githubusercontent.com/43805786/145708089-a9fdc569-914d-4203-a2e3-b2b9f1e1da6c.png)

**Reply by laurentabbal** [reply]
@charlesyo Effectivement, il faut lancer Edupyter en utilisant l'icone qui s'est installée sur le bureau. Ou alors, il faut aller dans le dossier dans lequel Edupyter est installé et double-cliquer sur `edupyter.exe`. Une fois lancé, Edupyter apparait en bas à droite à coté de l'horloge.
